MANIFEST.in

```
@@ -1,7 +1,8 @@
-include README.rst
-include LICENSE.txt
-include CODE_OF_CONDUCT.md
-include CONTRIBUTING.md
-include AUTHORS.md
-recursive-include pygmt/tests/data *
-recursive-include pygmt/tests/baseline *
+exclude AUTHORSHIP.md
+exclude MAINTENANCE.md
```

> Perhaps we should also remove `CODE_OF_CONDUCT.md` and `CONTRIBUTING.md` from source distributions?

Personally I would keep the CODE_OF_CONDUCT.md (5.3kb) but exclude CONTRIBUTING.md (21.3kb).

```suggestion
exclude CONTRIBUTING.md
exclude MAINTENANCE.md
```